src/combinatorics/young/young_diagram.lean

```diff
@@ -274,4 +274,42 @@ by { convert μ.transpose.row_len_anti j1 j2 hj; simp }
 
 end columns
 
+section row_lens
+/-! ### The list of row lengths of a Young diagram
+
+This section defines `μ.row_lens : list ℕ`, the list of row lengths of a Young diagram `μ`.
+  1. `row_lens_sorted` : It is weakly decreasing (list.sorted (≥)).
+  2. `row_lens_pos` : It is strictly positive.
+
+-/
+
+-- This belongs somewhere in the list namespace. Is it already in mathlib?
```

Comment:
Probably the best would be in `data/list/range.lean`, but if `list.sorted` isn't avaliable there, perhaps put these in `data/list/sort.lean`?

Reply:
Found it: there is `list.pairwise_lt_range` but no `list.pairwise_le_range`. Adding it seems to mean all of mathlib has to be recompiled (since `list` is so basic). Is there a way to speed it up?

Reply:
You can make the change, then push a commit to your branch; CI will run on it, and when it is done you can use `leanproject get-cache` to download the compiled oleans.